control0:
  ros__parameters:
    runner_cutter_control_params:
      camera_node_name: "camera0"
      laser_node_name: "laser0"
      tracking_laser_color: [0.15, 0.0, 0.0]
      burn_laser_color: [0.0, 0.0, 0.15]
      burn_time_secs: 1
camera0:
  ros__parameters:
    camera_control_params:
      camera_type: "lucid"
      camera_index: 0
      exposure_us: 20000.0
      gain_db: 1.0
      save_dir: "~/Pictures/runner-cutter-app"
      debug_frame_width: 640
laser0:
  ros__parameters:
    laser_control_params:
      dac_type: "helios"
      dac_index: 0
      fps: 30
      pps: 30000
      transition_duration_ms: 0.5
